Can't run kubernetes tests in Vagrant environment

# Description of problem

```
$ vagrant up ubuntu
[...]
$ vagrant ssh ubuntu
[...]
vagrant@ubuntu2004:~/go/src/github.com/kata-containers/tests$ make kubernetes
```

# Expected result

Runs kubernetes integration tests and I get results from those.

# Actual result

```
bash -f .ci/install_bats.sh
Install BATS from sources
.ci/install_bats.sh: line 16: pushd: /home/vagrant/go/src/github.com/bats-core/bats-core: No such file or directory
make: *** [Makefile:116: kubernetes] Error 1
```

--- a/.ci/install_bats.sh
+++ b/.ci/install_bats.sh
@@ -12,7 +12,7 @@ which bats && exit
 BATS_REPO="github.com/bats-core/bats-core"
 
 echo "Install BATS from sources"
-go get -d "${BATS_REPO}" || true
+GO111MODULE="auto" go get -d "${BATS_REPO}" || true
 pushd "${GOPATH}/src/${BATS_REPO}"
 sudo -E PATH=$PATH sh -c "./install.sh /usr"
 popd

